.gitignore

```diff
@@ -17,3 +17,6 @@ docs-local
 
 github_deploy_key
 sessions
+
+# use yarn, not npm, see README.md for details
+package-lock.json

```

Comment:
and package.json too, right?

Reply:
and docusaurus/package-lock.json ?

Reply:
This way Git will ignore files named `package-lock.json` no matter where they appear in the repo because the match pattern is not constrained by a path prefix. It's synonymous with `/**/package-lock.json`.